packages/react/src/combobox/root/ComboboxRootInternal.tsx

```diff
@@ -1203,17 +1207,12 @@ interface ComboboxRootProps<ItemValue> {
   actionsRef?: React.RefObject<ComboboxRootInternal.Actions>;
   /**
    * Callback fired when the user navigates the list and highlights an item.
-   * Passes the item and the type of navigation or `undefined` when no item is highlighted.
-   * - `keyboard`: The item was highlighted via keyboard navigation.
-   * - `pointer`: The item was highlighted via pointer navigation.
-   * - `none`: The item was highlighted via programmatic navigation.
+   * Receives the highlighted value (or `undefined` when none) and event details
+   * describing the navigation reason and highlighted index.
```

Comment:
Nitpick: Personally I'm not a fan of splitting JSDoc sentences into multiple lines.
What about AI? Won't it impact its ability to parse the source properly? 🤔 
If I remember correctly, I've seen @mapache-salvaje PR on `mui-x` tailoring JSDoc to be more digestible by machines as well as removing the new lines in sentences.